src/org/integr8ly/JiraUtils.groovy

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/groovy
+package org.integr8ly
+
+/**
+ * Requires the http_request plugin to be installed (https://plugins.jenkins.io/http_request)
+ * @param endpoint - Jira endpoint to send the request to
+ * @param httpMethod - HTTP method to be used for the request
+ * @param requestBody - HTTP request body
+ * @param jiraCredentials - Jira credentials to be used for authentication
+ * @param customHeaders - Additional headers to be used for the request
+ * @param validResponseCodes - Valid HTTP response codes
+ * @returns HTTP response
+ */
+def jiraApiRequest(String endpoint, String httpMethod = 'GET', String requestBody = null, String jiraCredentials, customHeaders = [], validResponseCodes = '100:399') {
+  def url = "https://issues.jboss.org/rest/api/2/${endpoint}"
```

Comment:
Should probably allow the host to be passed in.

Reply:
can the host be different here? We're doing the same with https://github.com/integr8ly/delorean-pipeline-library/blob/master/src/org/integr8ly/GitHubUtils.groovy#L25

Reply:
We have moved this into a public shared library that anyone could theoretically start using, so yes, it could be different if someone is using there own jira instance. GitHub is GitHub, but i guess it could be used for an enterprise github, but i really don't think we need worry about that.